### PR TITLE
Add an armv7l container

### PR DESCRIPTION
**Pull request by jjhelmus**
Add a Docker container for building linux-armv7l conda package.  Similar to `aarch64` the initial set of packages come from the `c4armv7l` channel on Anaconda.org which were build from the https://github.com/jjhelmus/conda4armv7l/ repository.

### CODE REVIEW

**Comment by jjhelmus** [comment]
.. and yes these packages will run on Raspberry Pi 2, 3 and 4s running Raspbian.

File: linux-anvil-armv7l/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+ENV LANG en_US.UTF-8
+ENV LANGUAGE=en_US.UTF-8
+
+# Add a timestamp for the build. Also, bust the cache.
+ADD http://worldclockapi.com/api/json/utc/now /opt/docker/etc/timestamp
+
+# configure yum and rpm for running on non-armv7l architectures
+RUN echo "armhfp" > /etc/yum/vars/basearch && \
+    echo "armv7hl" > /etc/yum/vars/arch && \
+    echo "armv7hl-redhat-linux-gpu" > /etc/rpm/platform
```
Comment (isuruf) [inline linux-anvil-armv7l/Dockerfile]: `gpu` -> `gnu`?

Comment (jjhelmus) [reply linux-anvil-armv7l/Dockerfile]: Yes, muscle memory must have let that sneak in.

**Comment by scopatz** [comment]
thanks!

**Pull request merged**